rust/src/api.rs

```diff
@@ -1,8 +1,18 @@
+use std::thread::sleep;
+use std::time::Duration;
+
+use crate::wallet;
 use anyhow::anyhow;
 use anyhow::Result;
-
 use flutter_rust_bridge::ZeroCopyBuffer;
 
+pub fn init_wallet() -> Result<String> {
+    println!("going to sleep");
```

Comment:
Do we have a logger already?

Reply:
I guess this ticket will cover it: https://github.com/itchysats/10101/issues/83

Reply:
no logger yet! we will disable `println` etc via clippy lint soon when we will add one.